screening/x-road.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "X-Road",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/nordic-institute/X-Road-development"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://x-road.global/resources"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "GDPR"
+    ],
+    "isLocIntlAdherent": "Yes",
+    "locIntlComplianceList": [
+      ""
```

Comment:
I believe this question is required if they answered `Yes` to `isLocIntlAdherent`

Reply:
Reached out to Ville, the contact person for  X-Road, will update once they come through with additional info on applicable international and domestic laws that the project complies with.